lib/logstash/inputs/syslog.rb

```diff
@@ -61,7 +80,9 @@ def register
     )
 
     @date_filter = LogStash::Filters::Date.new(
-      "match" => [ "timestamp", "MMM  d HH:mm:ss", "MMM dd HH:mm:ss", "ISO8601"]
+      "match" => [ "timestamp", "MMM  d HH:mm:ss", "MMM dd HH:mm:ss", "ISO8601"],
+      "locale" => @locale,
```

Comment:
plus the specs shows that it works correct with or without the `locale`/`timezone` config